kolena/workflow/test_case.py

```diff
@@ -93,6 +94,7 @@ def __init__(
     ):
         if type(self) == TestCase:
             raise Exception("<TestCase> must be subclassed.")
+        validate_not_blank(name)
```

Comment:
Does this prevent creating empty-name test cases via `TestCase.create`?

Reply:
Added this to TestCase.create as well and added matching tests